test/README.md

```diff
@@ -119,3 +119,19 @@ test infrastructure that wants to be agnostic to which `TimeSystem` is used in a
 test. When no `TimeSystem` is instantiated in a test, the `Event::GlobalTimeSystem`
 will lazy-initialize itself into a concrete `TimeSystem`. Currently this is
 `TestRealTimeSystem` but will be changed in the future to `SimulatedTimeSystem`.
+
+
+## Benchmark tests
+
+Envoy uses [Google Benchmark](https://github.com/google/benchmark/) for
+microbenchmarks. There are custom bazel rules, `envoy_cc_benchmark_binary` and
+`envoy_benchmark_test`, to execute them locally and in CI environments
+respectively. `envoy_benchmark_test` rules call the benchmark binary from a
+[script](https://github.com/envoyproxy/envoy/blob/master/bazel/test_for_benchmark_wrapper.sh)
+which runs the test quickly instead of rigourously. In order to collect
```

Comment:
Possible alternate wording suggestion:
which runs the benchmark with a minimal number of iterations and skipping expensive benchmarks to quickly verify that the binary is able to run to completion.